test/e2e/fixtures/then.go

```diff
@@ -55,3 +59,17 @@ func (t *Then) ExpectWorkflowList(listOptions metav1.ListOptions, block func(*te
 	block(t.t, wfList)
 	return t
 }
+
+func (t *Then) RunCli(args []string, block func(*testing.T, string)) *Then {
+	cmd := exec.Command("../../../dist/argo", args...)
```

Comment:
Can you please refactor the existing `argo(...)` command?

Reply:
Same as https://github.com/argoproj/argo/pull/1925#discussion_r365290283